packages/location-management/src/components/LocationTagView/Table.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { Table as AntTable, Menu, Dropdown, Button, Divider, Popconfirm, notification } from 'antd';
+import { MoreOutlined } from '@ant-design/icons';
+import { LocationTag } from '../../ducks/location-tags';
+import { getAccessToken } from '@onaio/session-reducer';
+import { useSelector } from 'react-redux';
+import { OpenSRPService } from '@opensrp/server-service';
+import { API_BASE_URL, LOCATION_TAG_DELETE, URL_LOCATION_TAG_EDIT } from '../../constants';
+import { Link } from 'react-router-dom';
+import { LocationTagDetailProps } from '../LocationTagDetail';
+
+export interface TableData extends LocationTag {
+  key: string;
+}
+
+export interface Props {
+  data: TableData[];
+  onViewDetails?: (locationUnit: LocationTagDetailProps) => void;
+}
+
+/** function to delete the record
+ *
+ * @param {object} record - The record to delete
+ * @param {string} accessToken - Access token to be used for requests
+ */
+export const onDelete = (record: LocationTag, accessToken: string) => {
+  const clientService = new OpenSRPService(
+    accessToken,
+    API_BASE_URL,
+    LOCATION_TAG_DELETE + record.id.toString()
+  );
+  clientService
+    .delete()
+    .then(() => notification.success({ message: 'Successfully Deleted!', description: '' }))
+    .catch((err: string) => notification.error({ message: `${err}`, description: '' }));
+};
+
+const Table: React.FC<Props> = (props: Props) => {
+  const { onViewDetails } = props;
+  const accessToken = useSelector((state) => getAccessToken(state) as string);
+
+  const columns = [
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      editable: true,
```

Comment:
Question?, what does editable do? looked at the documentation, could not find it?

Reply:
nice catch. we were using that before to toggle the state of editing in each row